icons/square-equal.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="18" height="18" x="3" y="3" rx="2" ry="2" />
+  <line x1="8" x2="16" y1="9" y2="9" />
+  <line x1="8" x2="16" y1="15" y2="15" />
```

Comment:
Let's use the same equals sign as in https://github.com/lucide-icons/lucide/pull/1032
```suggestion
  <path d="M7 10h10" />
  <path d="M7 14h10" />
```